libs/data-transforms/src/csv-to-track-segments/guessColumns.ts

```diff
@@ -1,24 +1,44 @@
-const GUESS_COLUMN_NAMES: [string, string[]][] = [
-  ['latitude', ['latitude', 'lat', 'location-lat']],
-  ['longitude', ['longitude', 'lng', 'lon', 'long', 'location-lng']],
-  ['timestamp', ['timestamp', 'time', 'date', 'datetime']],
+export type GuessColum = 'latitude' | 'longitude' | 'timestamp'
+
+export const LatitudeMatches = ['latitude', 'lat', 'location-lat', 'LAT', 'LATITUDE']
+export const LongitudeMatches = [
+  'longitude',
+  'lng',
+  'lon',
+  'long',
+  'location-lng',
+  'LNG',
+  'LON',
+  'LONG',
+  'LONGITUDE',
 ]
+export const TimestampMatches = ['timestamp', 'time', 'date', 'datetime']
+
+export const GUESS_COLUMN_DICT: Record<GuessColum, string[]> = {
+  latitude: LatitudeMatches,
+  longitude: LongitudeMatches,
+  timestamp: TimestampMatches,
+}
+
+export const GUESS_COLUMN_NAMES = [Object.entries(GUESS_COLUMN_DICT)]
 
-const guessColumns = (columns: string[] | undefined) => {
+export const guessColum = (col: GuessColum, options: string[]) => {
```

Comment:
Typo: colum -> column
```suggestion
export const guessColumn = (col: GuessColumn, options: string[]) => {
```